doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -585,7 +585,7 @@ Main OPNFV test tool candidates: Yardstick (TC 005), StorPerf
 
 The compute performance metrics are vnf.nfvi.perf.007 and infra.nfvi.perf.004,009.
 
-Each compute performance test should be performed with all vCPU of the node running multiple parallel workloads and the result is then normalized:
+The normalized result of the compute performance test requires all the vCPUs to run parallel workloads. You need to start as many vCPUs as needed to force all the physical CPUs on the node to run a workload. In this case, the result is normalized:
```

Comment:
Should this just say that all of the vCPUs should be running workloads? Some of the cores would however be reserved for  OSTK services. Please critique if the following states this clearly:
 ```suggestion
For normalized results, the compute performance test requires all of the physical cores and associated SMT threads (the total number of possible vCPUs) to run workloads.  You need to start as many VMs as needed to force all of the possible vCPUs on the node to run a workload. The results across multiple test case executions are, thus, normalized:
```

Reply:
> Should this just say that all of the vCPUs should be running workloads? Some of the cores would however be reserved for OSTK services. Please critique if the following states this clearly:

We tried to avoid adding the low-level description of the "total number of possible vCPUs" (i.e. physical cores and SMT threads) as that would raise questions regarding cases where there are no SMT threads, etc.

What do you think about changing this:
"all of the physical cores and associated SMT threads (the total number of possible vCPUs)"

to this:
"total number of vCPUs available for running workloads"

Does the "results across multiple test case executions" part cover all possible scenarios? Or is this relevant for single tests as well?

Reply:
@pgoyal01, I've added the correction, please check if it is OK.
Thanks!